cg/store/crud/read.py

```diff
@@ -1031,20 +1051,8 @@ def cases_to_analyse(self, workflow: Workflow = None, limit: int = None) -> list
             filter_functions=case_filter_functions,
```

Comment:
Would also be great to rename this function to `get_cases_to_analyse`

Reply:
Done